scripts/library/README.md

```diff
@@ -0,0 +1,30 @@
+# ukis-libraries-scripts
+
+This package contains a few scripts to build and test angular libraries.
+
+It is based on [depcheck](https://github.com/depcheck/depcheck) and [toposort](https://github.com/marcelklehr/toposort#readme)
+
+It uses the package scope `@dlr-eoc`, `main package.json` and the angular config file `angular.json`.
+
+
+- To get the list of all projects is reads the the angular config file (angular.json).
```

Comment:
To get the list of all projects it reads the angular config file (angular.json).

Reply:
done https://github.com/dlr-eoc/ukis-frontend-libraries/pull/90/commits/9fb3013f48b1a5e48c984dda50b34efbdd8a54ed.